### DISCUSSION TITLE
Route53 Plugin?

### DISCUSSION THREAD

**Discussion by burkeazbill** [Budibase/budibase#13031] [Ideas]
Has anyone done any work on Route53 automation using Budibase as the front-end?? I see that S3 is built-in, so some AWS type of work has been done here. My use-case would be the Creation/Removal of Hosted Zones (specifically subdomains) as well as the creation/update/removal of records in those zones.

Unfortunately, my searches for such have not yielded any usable results.

Thanks!

**Comment by andyburgessmd** [comment]
Possibly a nit niche to need a plugin. 
My suggestion would be to build something in AWS with API gateway and a lambda function to interact with R53 and then use the Rest API data source in Budibase to interact with the API gateway.
